project_management/turing_mybinder_role.md

```diff
@@ -0,0 +1,51 @@
+This cluster also hosts Hub23, a private BinderHub for the Turing research community.
+Maintaining this infrastructure requires technical skill and a small time commitment that are detailed below.
+
+It is important to the Binder Operating team that we are not anonymous GitHub handles to one another.
+Knowing who we are and why we're here helps us support each other and connect those who can help each other.
+Please do say hi and [introduce yourself](https://discourse.jupyter.org/t/introduce-yourself/17)! :smile:
+
+## Requirements
+
+- An account in the Turing's Azure tenancy
+- Access to the `turingmybinder` subscription (can be arranged with IT Services through Top Desk)
+- Knowledge of Kubernetes and Helm
+
+## Responsibilities
+
+Deployments are managed by CI in the [mybinder.org-deploy repo](https://github.com/jupyterhub/mybinder.org-deploy) and so day-to-day tasks are minimal and relatively non-disruptive.
+
+- Watch the [team-compass](https://github.com/jupyterhub/team-compass) for any relevant updates or opportunities for participation
+- Attend the [monthly team meetings](https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html) when able (does **not** have to be every month)
+- Be available in the [mybinder.org-deploy gitter chat](https://gitter.im/jupyterhub/mybinder.org-deploy) to liaise with the operating team should the cluster experience any technical difficulties
+- Be willing to co-work/pair programme with the mybinder.org operating teams to debug any issues with the cluster.
+  Particularly [Tim Head](https://github.com/betatim) who is the only other person with direct access to the subscription.
+- Monitor the `turingmybinder` subscription credits on Azure (under the Turing's tenancy) and [file requests for additional credits](https://turingcomplete.topdesk.net/tas/public/ssp/content/serviceflow?unid=fabf0809069f42a5a36c61d677da08fa&openedFromService=true) using [the template](credit_requests/turing_mybinder_azure_credit_request_template.md)
+  - **N.B.:** Now that Hub23 and the Turing Federation Hub share the same infrastructure/subscription, this template and further credit requests may need updating.
+    For example, there are now 2 key vaults and 2 DNS zones deployed in the subscription, alongside 1 Kubernetes cluster and 1 Azure Container Registry.
+- Merge Pull Requests from [HelmUpgradeBot](https://github.com/HelmUpgradeBot) on the [hub23-deploy repo](https://github.com/alan-turing-institute/hub23-deploy).
+  These PRs keep the Hub23 deployment up-to-date with helm chart releases from BinderHub and `ingress-nginx`.
+  - It is up to the person in the role if they would like to continue the development of Hub23, however, this is beyond the scope of "minimal" responsibilities.
+    [@sgibson91](https://github.com/sgibson91) is happy to discuss further with any interested parties.
+
```

Comment:
Few extra questions: does this role "report" to anyone in the Turing? Will they be asked to update people internally on this piece of infrastructure? This may be the case if they're filing additional requests for credits. Also, is there a potential time limit on funding? 

The answer to all these might be "not relevant" but might be worth thinking about that side of things so it's not a surprise to anyone who takes this on.

Reply:
Partially addressed in bbb74d0

Kirstie would be the person to report to as she's listed as the PI on the credit requests. @KirstieJane some really useful input from you here would be how you would prefer to be kept updated by this person and how frequently, we've been doing this pretty informally/ad-hoc so far :) I don't think it needs to be a very frequent though as major changes aren't made often.